### DISCUSSION TITLE
naming next version v2.0.0 may lead to confusion

### DISCUSSION THREAD

**Discussion by gregnuj** [classic-terra/core#196] [General]
while I generally agree a major upgrade deserves a major version bump, please note the tfl versioning used 1.x.x for classic and 2.x.x for terra2, I can imagine some folks will get confused and be downloading the terra2 tfl repo at v2.0.0
https://github.com/terra-money/core/releases/tag/v2.0.0

**Comment by ZaradarBH** [comment]
We follow SemVer. SemVer dictates that the next version number will be 2.0.0 due to the change in the underlying Cosmos SDK dependencies which will cause breakage for L2 apps, making it a major version bump.
